code/modules/antagonists/heretic/items/heretic_armor.dm

```diff
@@ -28,10 +27,10 @@
 	var/hood_up = FALSE
 
 /datum/armor/cultrobes_eldritch
-	melee = 50
-	bullet = 50
-	laser = 50
-	energy = 50
+	melee = 30
+	bullet = 30
+	laser = 30
+	energy = 30
```

Comment:
This is indeed worse than secoff armor
```js
/datum/armor/suit_armor
	melee = 35
	bullet = 30
	laser = 30
	energy = 40
```
```js
/datum/armor/head_helmet
	melee = 35
	bullet = 30
	laser = 30
	energy = 40
```
Both vests and helmets have better stats